code/modules/mob/living/simple_animal/hostile/pulse_demon/pulsedemon.dm

```diff
@@ -397,24 +434,20 @@
 
 // This too
 /mob/living/simple_animal/hostile/pulse_demon/proc/drainAPC(var/obj/machinery/power/apc/current_apc)
-	// Was too pitiful at normal level, this should now allow a maximum of 400kw from the best power cells, just below 600kw from best SME
-	max_can_absorb = current_apc.cell.maxcharge * 10
+	//max_can_absorb = current_apc.cell.maxcharge * 10 //draining APC batteries has no upper limit on maxpower due to uhhh galvanic isolation
 	var/amount_to_drain = charge_absorb_amount
 	// Cap conditions
 	if(current_apc.cell.charge <= amount_to_drain)
 		amount_to_drain = current_apc.cell.charge
-	if(maxcharge <= max_can_absorb && charge >= maxcharge)
-		maxcharge += amount_to_drain
-	else if(charge >= maxcharge)
-		amount_to_drain = 0
+	maxcharge += amount_to_drain * 2 //multiplier to balance the pitiful powercells in APCs
```

Comment:
instead of just a magic number, why not a define at the top of the file:
`#define APC_CHARGE_MULTIPLIER 2`
then here it would be `maxcharge += amount_to_drain * APC_CHARGE_MULTIPLIER`
This makes it obvious what it does even without a comment, and easy to maintain if someone wants to change the number at the top.

Reply:
what the hell this wasn't even supposed to be in this PR, this should be in the other because BALANCE